contracts/SwapHelperLib.sol

```diff
@@ -4,6 +4,7 @@ pragma solidity =0.8.7;
 import "@uniswap/v2-periphery/contracts/interfaces/IUniswapV2Router02.sol";
 import "@uniswap/v2-periphery/contracts/interfaces/IUniswapV2Router01.sol";
 import "@zetachain/protocol-contracts/contracts/zevm/interfaces/IZRC20.sol";
+import "@zetachain/protocol-contracts/contracts/zevm/SystemContract.sol";
```

Comment:
If it's imported, `npx hardhat compile` generates `typechain-types` for the system contract. These types are imported by some TS files in `helpers`. If there is no import and let's say you delete `typechain-types`, the build will fail.

Reply:
in that case you should import it here don't you think? https://github.com/zeta-chain/toolkit/blob/main/contracts/TestSystemContract.sol

Reply:
Good point! I had to remove `SystemContractErrors`, because it was throwing an error:

```
DeclarationError: Identifier already declared.
 --> contracts/TestSystemContract.sol:8:1:
  |
8 | interface SystemContractErrors {
  | ^ (Relevant source part starts here and spans across multiple lines).
Note: The previous declaration is here:
 --> contracts/TestSystemContract.sol:6:1:
  |
6 | import "@zetachain/protocol-contracts/contracts/zevm/SystemContract.sol";
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

But I don't think it's a big deal.

https://github.com/zeta-chain/toolkit/pull/47/commits/4a6c68800251b26ef26a3d7bf1347ad4a2e6809e